lib/client.js

```diff
@@ -252,7 +247,7 @@ var Client = function(options) {
     return this; // shortcut handshake
   }
 
-  setupTracking(this)
+  Tracking.setupTracking(this);
```

Comment:
Would it make more sense to call this `Tracking.setup`?

Reply:
Sure! Done.